doc/usage.rst

```diff
@@ -204,21 +207,34 @@ Then, a :any:`Driver` needs to be created on the `Target`::
   >>> sd = SerialDriver(t, name=None)
 
 
-As the `SerialDriver` declares a binding to a SerialPort, the target binds it
+As the :any:`SerialDriver` declares a binding to a :any:`SerialPort`, the target binds it
 to the resource created above::
 
   >>> sd.port
   RawSerialPort(target=Target(name='example', env=None), name=None, state=<BindingState.bound: 1>, avail=True, port='/dev/ttyUSB0', speed=115200)
   >>> sd.port is rsp
   True
 
-Before the driver can be used, it needs to be activated::
+Driver Activation
+^^^^^^^^^^^^^^^^^
+Before the driver can be used, it needs to be activated.
+On driver activation, labgrid makes sure that all resources needed by the
+driver are available::
```

Comment:
> makes sure that all resources needed by the driver are available

What are you trying to say here?
Because a few sentences below there is refutal of `labgrid's ability to making sure...`.
FWIW here is a suggestion :)

```suggestion
Before the bounded driver can be used, it needs to be activated.
An activation is an act of the driver consuming the underlying resource on a system level. 

If we use a car analogy, binding is the process of screwing the car together from parts and activation is igniting the engine.

I.e. If a `SerialDriver` is bound to a `RawSerialPort` representing `/dev/ttyUSB0`. An external application could like `minicom` could consume the `/dev/ttyUSB0`. Once the driver is activated, that is no longer possible. ::
```

Reply:
I like tha analogy!